openlr/helpers.hpp

```diff
@@ -2,15 +2,34 @@
 
 #include "openlr/graph.hpp"
 #include "openlr/openlr_model.hpp"
+#include "openlr/score_types.hpp"
 
 #include "geometry/point2d.hpp"
 
+#include <cstdint>
+#include <string>
 #include <type_traits>
 
 namespace openlr
 {
 class RoadInfoGetter;
 
+// This class is used to get correct points for further bearing calculations.
+// Depending on |isLastPoint| it either calculates those points straightforwardly
+// or reverses directions and then calculates.
+class BearingPointsSelector
+{
+public:
+  BearingPointsSelector(uint32_t bearDistM, bool isLastPoint);
+
+  m2::PointD GetBearingStartPoint(Graph::Edge const & e) const;
```

Comment:
`GetBearingStartPoint` => `GetStartPoint`

Reply:
GetStartPoint() and GetEndPoint()